website/src/data/proto-tools.tsx

```diff
@@ -247,4 +247,13 @@ export const THIRD_PARTY_TOOLS: Record<string, ProtoTool> = {
 		pluginType: 'toml',
 		repoUrl: 'https://github.com/stk0vrfl0w/proto-toml-plugins/blob/main/plugins/zig.toml',
 	},
+	'zig-plugin': {
```

Comment:
I would like the tool identifier to be `zig`.
However, `zig` already exists. Therefore, the current data structure does not allow that. 🤔

Perhaps we should allow multiple plugins to be associated with a single tool ID in the documentation.

But now I do this.

Reply:
Yeah I'll look into supporting an array.